Errors building

Used

```
git clone https://github.com/alicevision/AliceVision.git --recursive
mkdir build && cd build
cmake -DALICEVISION_BUILD_DEPENDENCIES=ON -DCMAKE_INSTALL_PREFIX=$PWD/../install ../AliceVision
make -j10
```

Mid stream make received:

```
CMake Error at external/src/boost-stamp/download-boost.cmake:170 (message):
  Each download failed!

    error: downloading 'https://boostorg.jfrog.io/artifactory/main/release/1.84.0/source/boost_1_84_0.tar.bz2' failed
          status_code: 22
          status_string: "HTTP response code said error"
          log:
          --- LOG BEGIN ---
            Trying 18.214.194.113:443...

  Connected to boostorg.jfrog.io (18.214.194.113) port 443 (#0)

  ALPN: offers h2

  ALPN: offers http/1.1

  TLSv1.0 (OUT), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.3 (OUT), TLS handshake, Client hello (1):

  [512 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.3 (IN), TLS handshake, Server hello (2):

  [112 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Certificate (11):

  [3701 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Server key exchange (12):

  [300 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Server finished (14):

  [4 bytes data]

  TLSv1.2 (OUT), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (OUT), TLS handshake, Client key exchange (16):

  [37 bytes data]

  TLSv1.2 (OUT), TLS header, Finished (20):

  [5 bytes data]

  TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):

  [1 bytes data]

  TLSv1.2 (OUT), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (OUT), TLS handshake, Finished (20):

  [16 bytes data]

  TLSv1.2 (IN), TLS header, Finished (20):

  [5 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Finished (20):

  [16 bytes data]

  SSL connection using TLSv1.2 / ECDHE-RSA-AES256-GCM-SHA384

  ALPN: server accepted http/1.1

  Server certificate:

   subject: CN=*.jfrog.io
   start date: Jan 17 00:00:00 2024 GMT
   expire date: Feb 16 23:59:59 2025 GMT
   subjectAltName: host "boostorg.jfrog.io" matched cert's "*.jfrog.io"
   issuer: C=US; O=DigiCert Inc; OU=www.digicert.com; CN=GeoTrust TLS RSA CA G1
   SSL certificate verify result: self-signed certificate in certificate chain (19), continuing anyway.

  TLSv1.2 (OUT), TLS header, Supplemental data (23):

  [5 bytes data]

  GET /artifactory/main/release/1.84.0/source/boost_1_84_0.tar.bz2 HTTP/1.1

  Host: boostorg.jfrog.io

  User-Agent: curl/7.86.0

  Accept: */*

  

  TLSv1.2 (IN), TLS header, Supplemental data (23):

  [5 bytes data]

  Mark bundle as not supporting multiuse

  HTTP/1.1 302 Moved Temporarily

  Date: Thu, 09 Jan 2025 11:11:25 GMT

  Content-Type: text/html

  Content-Length: 138

  Connection: keep-alive

  Location: https://landing.jfrog.com/reactivate-server/boostorg

  

  Ignoring the response-body

  [138 bytes data]

  Connection #0 to host boostorg.jfrog.io left intact

  Issue another request to this URL:
  'https://landing.jfrog.com/reactivate-server/boostorg'

    Trying 18.232.172.199:443...

  Connected to landing.jfrog.com (18.232.172.199) port 443 (#1)

  ALPN: offers h2

  ALPN: offers http/1.1

  TLSv1.0 (OUT), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.3 (OUT), TLS handshake, Client hello (1):

  [512 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.3 (IN), TLS handshake, Server hello (2):

  [112 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Certificate (11):

  [3672 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Server key exchange (12):

  [300 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Server finished (14):

  [4 bytes data]

  TLSv1.2 (OUT), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (OUT), TLS handshake, Client key exchange (16):

  [37 bytes data]

  TLSv1.2 (OUT), TLS header, Finished (20):

  [5 bytes data]

  TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):

  [1 bytes data]

  TLSv1.2 (OUT), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (OUT), TLS handshake, Finished (20):

  [16 bytes data]

  TLSv1.2 (IN), TLS header, Finished (20):

  [5 bytes data]

  TLSv1.2 (IN), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Finished (20):

  [16 bytes data]

  SSL connection using TLSv1.2 / ECDHE-RSA-AES256-GCM-SHA384

  ALPN: server accepted http/1.1

  Server certificate:

   subject: CN=*.jfrog.com
   start date: May 20 00:00:00 2024 GMT
   expire date: Jun  6 23:59:59 2025 GMT
   subjectAltName: host "landing.jfrog.com" matched cert's "*.jfrog.com"
   issuer: C=US; O=DigiCert Inc; OU=www.digicert.com; CN=GeoTrust TLS RSA CA G1
   SSL certificate verify result: self-signed certificate in certificate chain (19), continuing anyway.

  TLSv1.2 (OUT), TLS header, Supplemental data (23):

  [5 bytes data]

  GET /reactivate-server/boostorg HTTP/1.1

  Host: landing.jfrog.com

  User-Agent: curl/7.86.0

  Accept: */*

  

  TLSv1.2 (IN), TLS header, Supplemental data (23):

  [5 bytes data]

  Mark bundle as not supporting multiuse

  HTTP/1.1 409 Conflict

  Date: Thu, 09 Jan 2025 11:11:27 GMT

  Content-Type: text/html

  Content-Length: 144

  Connection: keep-alive

  Strict-Transport-Security: max-age=31536000; includeSubDomains

  The requested URL returned error: 409

  Closing connection 1

  TLSv1.2 (OUT), TLS header, Unknown (21):

  [5 bytes data]

  TLSv1.2 (OUT), TLS alert, close notify (256):

  [2 bytes data]

  

          --- LOG END ---
```

Last line of console out before it stopped:

```
[ 67%] Completed 'opengv'
[ 67%] Built target opengv
make: *** [Makefile:91: all] Error 2
```

--- a/src/cmake/Dependencies.cmake
+++ b/src/cmake/Dependencies.cmake
@@ -620,7 +620,7 @@ if(AV_BUILD_BOOST)
     endif()
     
     ExternalProject_Add(${BOOST_TARGET}
-        URL https://boostorg.jfrog.io/artifactory/main/release/1.84.0/source/boost_1_84_0.tar.bz2
+        URL https://archives.boost.io/release/1.84.0/source/boost_1_84_0.tar.bz2
         URL_HASH MD5=9dcd632441e4da04a461082ebbafd337
         DOWNLOAD_DIR ${BUILD_DIR}/download/boost
         PREFIX ${BUILD_DIR}
@@ -630,15 +630,15 @@ if(AV_BUILD_BOOST)
         SOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR}/boost
         BINARY_DIR ${BUILD_DIR}/boost_build
         INSTALL_DIR ${CMAKE_INSTALL_PREFIX}
-        CONFIGURE_COMMAND 
-            cd <SOURCE_DIR> && 
+        CONFIGURE_COMMAND
+            cd <SOURCE_DIR> &&
             ./bootstrap.${SCRIPT_EXTENSION} --prefix=<INSTALL_DIR> --with-libraries=atomic,container,date_time,exception,graph,iostreams,json,log,math,program_options,regex,serialization,system,test,thread,stacktrace,timer
-        BUILD_COMMAND 
-            cd <SOURCE_DIR> && 
+        BUILD_COMMAND
+            cd <SOURCE_DIR> &&
             ./b2 --prefix=<INSTALL_DIR> variant=${DEPS_CMAKE_BUILD_TYPE_LOWERCASE} cxxstd=17 link=shared threading=multi -j8
-        INSTALL_COMMAND 
-            cd <SOURCE_DIR> && 
-            ./b2 variant=${DEPS_CMAKE_BUILD_TYPE_LOWERCASE}  cxxstd=17 link=shared threading=multi install
+        INSTALL_COMMAND
+            cd <SOURCE_DIR> &&
+            ./b2 variant=${DEPS_CMAKE_BUILD_TYPE_LOWERCASE} cxxstd=17 link=shared threading=multi install
         DEPENDS ${ZLIB_TARGET}
     )
 

